.github/workflows/ci.yml

```diff
@@ -0,0 +1,69 @@
+name: CI
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - "*.*.*"
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.ivy2
+            ~/.sbt
+          key: ${{ runner.os }}-sbt-${{ hashFiles('**/build.sbt') }}
```

Comment:
Should the files in `.ivy2` be part of the key as well?
Is there any benefit to having the runner.os in the key? It seems we're only running this in ubuntu-20.04

Reply:
I took that from examples for cache action. I assumed that its useful when we upgrade the os at some point in the future?